hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/utils/Pipelines.java

```diff
@@ -315,14 +315,14 @@ public static DataStream<HoodieRecord> rowDataToHoodieRecord(Configuration conf,
    * @param dataStream         The input data stream
    * @return the stream write data stream pipeline
    */
-  public static DataStream<Object> hoodieStreamWrite(Configuration conf, int defaultParallelism, DataStream<HoodieRecord> dataStream) {
+  public static DataStream<Object> hoodieStreamWrite(Configuration conf, int defaultParallelism, DataStream<HoodieRecord> dataStream, String sinkOperatorNameSuffix) {
     if (OptionsResolver.isBucketIndexType(conf)) {
```

Comment:
The `sinkOperatorNameSuffix` can be resolved in the `conf`, there is no need to pass it around explicitly.

Reply:
yes，but sinkOperatorNameSuffix is not a configuration, so it is best to pass it explicitly :)

Reply:
and may have multi-thread conflicts